### ISSUE TITLE
error on user module

### ISSUE THREAD

**Issue by mlarac** [phpservermon/phpservermon#573]
Fatal error: Declaration of psm\Module\User\Controller\UserController::run() must be compatible with psm\Module\AbstractController::run($action = NULL) in /home/monitorredes/monitor.redessur.cl/src/psm/Module/User/Controller/UserController.php on line 36

**Comment by TimZ99** [comment]
Fixed with the next update. https://github.com/phpservermon/phpservermon/blob/fc4ffd6b2f6611cb983d2adf57409d53e4e17f48/src/psm/Module/User/Controller/UserController.php#L51

**Comment by jimmyg2times** [comment]
I'm using v3.2.0 just recently installed.  Getting this error:
Fatal error: Declaration of psm\Module\User\Controller\UserController::run() must be compatible with psm\Module\AbstractController::run($action = NULL) in /var/www/html/servermon/src/psm/Module/User/Controller/UserController.php on line 36

Edit:  PHP Version:  PHP 7.3.13

**Comment by TimZ99** [comment]
Hi, don't wanna be rude, but did you even read this issue... 

It probably took you longer to write this issue than to read it.
It's only 2 comments long! One is the problem and the other one is the solution...

Please see the [release page](https://github.com/phpservermon/phpservermon/releases/tag/v3.4.5) for the latest release. 3.2.0 is pretty outdated.

**Comment by jimmyg2times** [comment]
It's ok if you want to be rude.  I'm a friggin idiot reading too fast and missing.  Yep, I'm that dumbass.  I'm sorry Tim, I don't normally do this and will slow down in the future.  READ.  :)  

Many apologies....

**Comment by TimZ99** [comment]
Haha it’s okay 🙈😜
________________________________
Van: jimmyg2times <notifications@github.com>
Verzonden: Thursday, December 19, 2019 7:41:56 PM
Aan: phpservermon/phpservermon <phpservermon@noreply.github.com>
CC: Tim <tim.zandbergen@outlook.com>; State change <state_change@noreply.github.com>
Onderwerp: Re: [phpservermon/phpservermon] error on user module (#573)


It's ok if you want to be rude. I'm a friggin idiot reading too fast and missing. Yep, I'm that dumbass. I'm sorry Tim, I don't normally do this and will slow down in the future. READ. :)

Many apologies....

**Comment by jimmyg2times** [comment]
FWIW - I made the change and MAGICALLY it works!  Ugh I'm still a dumbass tho.  I can't believe I skimmed right over and just posted.  Ahh the n00b in me.  Embarrassing.

**Comment by harveydobson** [comment]
> Hi, don't wanna be rude, but did you even read this issue...
> 
> It probably took you longer to write this issue than to read it.
> It's only 2 comments long! One is the problem and the other one is the solution...
> 
> Please see the [release page](https://github.com/phpservermon/phpservermon/releases/tag/v3.4.5) for the latest release. 3.2.0 is pretty outdated.

Hi All, 

I'd just like to say, I have also encountered the issue. I appreciate you pointing us in the right direction, as I was about to post a 'me too' but only because I figured that the issue that was resolved had been accidentally un-resolved in someway. 

FYI I discovered this software through the official website https://www.phpservermonitor.org/download/ in which, it advertises the latest release as being 3.2.0, both on the downloads page and in recent posts. 

It only directs people to the github for "development snapshots".

Perhaps the website should link to the releases page of the github for the latest download https://github.com/phpservermon/phpservermon/releases

It's a great piece of software, wouldn't want people to disregard it thinking that it is buggy. 

H.
